<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmallBASIC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sbasic.qrc">
    <normaloff>:/images/smallbasic.png</normaloff>:/images/smallbasic.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="AnsiWidget" name="ansiWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewWindow"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHomePage"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionFindAgain"/>
    <addaction name="separator"/>
    <addaction name="actionErrorConsole"/>
    <addaction name="actionProgramSource"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>&amp;Bookmarks</string>
    </property>
    <addaction name="actionBookmarkProgram"/>
    <addaction name="actionViewBookmarks"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionBreak"/>
   <addaction name="actionHome"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open File...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About SmallBASIC</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>&amp;Find</string>
   </property>
  </action>
  <action name="actionFindAgain">
   <property name="text">
    <string>Find &amp;Again</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Prefere&amp;nces</string>
   </property>
  </action>
  <action name="actionHomePage">
   <property name="text">
    <string>&amp;Home page</string>
   </property>
  </action>
  <action name="actionBreak">
   <property name="icon">
    <iconset resource="sbasic.qrc">
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>Break</string>
   </property>
   <property name="toolTip">
    <string>Halt current program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="sbasic.qrc">
     <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
   <property name="toolTip">
    <string>Reload current program</string>
   </property>
  </action>
  <action name="actionProgramSource">
   <property name="text">
    <string>Program &amp;Source</string>
   </property>
  </action>
  <action name="actionErrorConsole">
   <property name="text">
    <string>Error &amp;Console</string>
   </property>
  </action>
  <action name="actionBookmarkProgram">
   <property name="text">
    <string>Bookmark This Program</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="text">
    <string>&amp;New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionViewBookmarks">
   <property name="text">
    <string>&amp;View &amp;Bookmarks</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="sbasic.qrc">
     <normaloff>:/images/home.png</normaloff>:/images/home.png</iconset>
   </property>
   <property name="text">
    <string>home</string>
   </property>
   <property name="toolTip">
    <string>Run the home program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="sbasic.qrc">
     <normaloff>:/images/jump.png</normaloff>:/images/jump.png</iconset>
   </property>
   <property name="text">
    <string>start</string>
   </property>
   <property name="toolTip">
    <string>Start the entered program</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="sbasic.qrc">
     <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>back</string>
   </property>
   <property name="toolTip">
    <string>Go back</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="sbasic.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
   <property name="text">
    <string>next</string>
   </property>
   <property name="toolTip">
    <string>Go forward</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="focusUrl">
   <property name="text">
    <string>focusUrl</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnsiWidget</class>
   <extends>QWidget</extends>
   <header>ansiwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sbasic.qrc"/>
 </resources>
 <connections/>
</ui>
